suite/tests/CMakeLists.txt

```diff
@@ -3657,7 +3657,7 @@ if (BUILD_CLIENTS)
     set(tool.drcov.fib_expectbase "tool.drcov.fib")
     DynamoRIO_get_full_path(tool.drcov.fib_postcmd drcov2lcov "${location_suffix}")
 
-    if (UNIX AND NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
+    if (UNIX)
       # Test an app that executes a pipe syscall for i#5981.
       torunonly_ci(tool.drcov.eintr linux.eintr drcov linux/eintr.c "" "" "")
```

Comment:
Update the PR description (will become commit message) to also say that this test is not added to RUNS_ON_QEMU yet.

Reply:
OK. I will update the PR description.